backend/main.py

```diff
@@ -1,48 +1,68 @@
 """
 Main flask script. Defines the '/recommendation/' route and parses input. 
 """
-# TODO: more robust query evaluation
+import os
+
 from flask import Flask, request, jsonify
+
 from recommendation.RecommendationResource import initModel, recommendTasks
 from data_collection.scraper_scheduler import initScheduler
 from data_collection.web_scraper import scrapeARC, get_next_n_events
+import db
 
 
-app = Flask(__name__)
-app.before_first_request(initScheduler)
-app.before_first_request(initModel)
-app.before_first_request(scrapeARC)
+def create_app() -> Flask:
+    """
+    Create and configure the app. When run using 'flask run' from the terminal, this 
+    function will be called to get the Flask application instance.
+    """
 
+    app = Flask(__name__, instance_relative_config=True)
 
-@app.route("/recommendation/", methods=['GET'])
-def recommendation():
-    """
-    GET route that takes two parameters, hands them off to recommendation
-    algorithm, and sends back the result. 
-    Params:
-        num_resources: int
-        query: [float]
+    # Set database file path
+    app.config.from_mapping(
+        DATABASE="./flaskr.sqlite"),
```

Comment:
This line should be changed to: 
`DATABASE="./flaskr.sqlite"`

instead of: `DATABASE="./flaskr.sqlite"),`

Reply:
Good catch! I modified it on Github when I saw the absolute path, and I must have missed that. I won’t be able to fix it until late tonight, if you want to change it here and commit then merge the pull that’d be awesome. If not I can do it later!

Reply:
I made the change! :) I will merge it now for you! Great job!!